### DISCUSSION TITLE
Hook into field updates?

### DISCUSSION THREAD

**Discussion by sector101010** [mdelobelle/metadatamenu#502] [Q&A]
Hi!

Thanks for the great plugin! 
Is it somehow possible to hook into the update of a specific property/field, to trigger a custom function? I'd like to update the `done_date` property of my `task` fileClass whenever the `status` property is set to "Done".

**Comment by mdelobelle** [comment]
Hi thanks for the feedback!
That's a very good idea. I could add "callback function" in the settings, to do things once the value of the field has been updated with metadata menu.
This would be for (very) advanced users though

WDYT?

**Comment by sector101010** [comment]
Hey - that sounds like a great solution! The callback function would be triggered whenever you update a field from the mdm modal or from the command function in a dataviewjs codeblock `f(dv, p, ...)`, correct?

In my case, I'm using [obsidian-custom-js](https://github.com/saml-dev/obsidian-custom-js) so it would be great if the `customJS` global would be available to reference the desired function. But I guess that's not a problem..
